server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestClearIndicesCacheAction.java

```diff
@@ -99,9 +99,9 @@ public static ClearIndicesCacheRequest fromRequest(final RestRequest request, Cl
     }
 
     public static class Fields {
-        public static final ParseField QUERY = new ParseField("query", "filter", "filter_cache");
-        public static final ParseField REQUEST = new ParseField("request", "request_cache");
-        public static final ParseField FIELDDATA = new ParseField("fielddata", "field_data");
+        public static final ParseField QUERY = new ParseField("query");
+        public static final ParseField REQUEST = new ParseField("request");
+        public static final ParseField FIELDDATA = new ParseField("fielddata");
```

Comment:
at this point using ParseField s for parameters can be replaced with just strings.

Reply:
Nice catch!
Do you wanna take it over or should I?

Reply:
all yours if you have time for it ;)